tide_core.install

```diff
@@ -1331,3 +1331,62 @@ function tide_core_update_8054() {
     $config->set('cardinality', -1)->save();
   }
 }
+
+/**
+ * Installs new statistics_grid paragraph type.
+ */
+function tide_core_update_8055() {
+  $configs = [
+    'paragraphs.paragraphs_type.statistics_grid' => 'paragraphs_type',
```

Comment:
only the definitions of `storage` move to tide_core. 
other settings should stay with dependent modules.

Reply:
Changes made